lib/main.dart

```diff
@@ -1,16 +1,18 @@
-import 'dart:convert';
-
 import 'package:flutter/material.dart';
-import 'package:shared_preferences/shared_preferences.dart';
+import 'package:todo/provider.dart';
 import 'package:todo/todo.dart';
 
 void main() {
-  runApp(MaterialApp(
-    home: const MyHomePage(),
-    debugShowCheckedModeBanner: false,
-    theme: ThemeData(
-      colorScheme: const ColorScheme.light(
-        primary: Colors.black,
+  WidgetsFlutterBinding.ensureInitialized();
+  runApp(Provider(
```

Comment:
missing trailing `comma` for formatting to look nicer.

Reply:
fixed